Source/ACE.Server/Managers/PlayerManager.cs

```diff
@@ -27,11 +27,18 @@ public static class PlayerManager
         private static readonly Dictionary<uint, Player> onlinePlayers = new Dictionary<uint, Player>();
         private static readonly Dictionary<uint, OfflinePlayer> offlinePlayers = new Dictionary<uint, OfflinePlayer>();
 
+        public static readonly long DefaultPlayerSaveInterval = 5;
```

Comment:
When I was thinking about this, I was thinking the unit of measurement would be seconds.

Reply:
I'll add a suffix indicating it's minutes.